doc/calcurse.1.txt

```diff
@@ -30,519 +30,714 @@
+*--format-recur-event* 'format'::
 
-*--filter-end-before* <date>::
-  Only include items that end before a given date.
+*--format-todo* 'format'::
+  The 'format' argument is a string composed of printf-style format
+  specifiers, which are replaced as explained below, and ordinary characters,
+  which are copied to stdout without modification. Each option has a default
+  'format' string which is used when the option is not given. Default strings
+  are described in <<_default_format_strings,Default Format Strings>>.
++
+'Note': Use of a format option requires explicit formatting of field
+separation and line spacing.
 
-*--filter-end-range* <range>::
-  Only include items with an end date that falls within a given range. A range
-  consists of a start date and an end date, separated by a comma.
+.Format specifiers
 
-*--filter-priority* <priority>::
-  Only include items with a given priority.
+Each specifier is introduced by a +%+ followed by a character which tells what to
+print. The available specifiers depend on the item type. Times are printed as
+hours and minutes (+hh:mm+) unless otherwise noted; time formats can be
+changed with <<_extended_format_specifiers,extended specifiers>>.
 
-*--filter-completed*::
-  Only include completed TODO items.
+.+appointments+
 
-*--filter-uncompleted*::
-  Only include uncompleted TODO items.
-
-Formatting Options
-~~~~~~~~~~~~~~~~~~
-
-*--format-apt* <format>::
-  Specify a format to control the output of appointments in non-interactive
-  mode. See the 'FORMAT STRINGS' section for detailed information on format
-  strings.
-
-*--format-recur-apt* <format>::
-  Specify a format to control the output of recurrent appointments in
-  non-interactive mode. See the 'FORMAT STRINGS' section for detailed
-  information on format strings.
-
-*--format-event* <format>::
-  Specify a format to control the output of events in non-interactive mode. See
-  the 'FORMAT STRINGS' section for detailed information on format strings.
-
-*--format-recur-event* <format>::
-  Specify a format to control the output of recurrent events in non-interactive
-  mode. See the 'FORMAT STRINGS' section for detailed information on format
-  strings.
-
-*--format-todo* <format>::
-  Specify a format to control the output of todo items in non-interactive mode.
-  See the 'FORMAT STRINGS' section for detailed information on format strings.
-
-Format strings
---------------
-
-Format strings are composed of printf()-style format specifiers -- ordinary
-characters are copied to stdout without modification. Each specifier is
-introduced by a *%* and is followed by a character which specifies the field to
-print. The set of available fields depends on the item type.
-
-Format specifiers for appointments
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-*s*::
-  Print the start time of the appointment as UNIX time stamp
-*S*::
-  Print the start time of the appointment using the *hh:mm* format
-*d*::
+[horizontal]
```

Comment:
These commands generate labelled tables in the generated man page which look quite odd...

Reply:
Corrected.